FaliJedan/FaliJedan.Domain/Repositories/Implementations/EventRepository.cs

```diff
@@ -24,17 +24,18 @@ public bool AddEvent(Event eventToAdd)
             if (
                 eventToAdd.CurrentNumberOfPlayers < 1 || 
                 eventToAdd.TargetNumberOfPlayers < 2 ||
-                eventToAdd.DateOfEvent == null || 
+                eventToAdd.TargetNumberOfPlayers > eventToAdd.CurrentNumberOfPlayers ||
```

Comment:
In my opinion this should be >= instead of > considering the fact that if they're equal, users don't really need any more players ^^